src/components/BottomSheet.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useRef } from 'react'
+import { Animated, Easing, Pressable, StyleSheet, useWindowDimensions } from 'react-native'
+
+// source: https://stackoverflow.com/questions/39117599/how-to-slide-view-in-and-out-from-the-bottom-in-react-native
+
+const BottomSheet = (
+  { children, show, height, onOuterClick }:
+  { children: React.ReactNode, show: boolean, height: number, onOuterClick?: () => void }) => {
+
+  const useAnimatedBottom = (show: boolean, height: number) => {
+    const animatedValue = useRef(new Animated.Value(0))
+    const bottom = animatedValue.current.interpolate({ inputRange: [0, 1], outputRange: [-height, 0] })
```

Comment:
- [ ] Create a hooks folder
- [ ] Add hooks folder to alias paths in `tsconfig.json`
- [ ] Add this to its own file in hooks.